lib/generators/action_view/component/component_generator.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module ActionView
```

Comment:
My rationale for not adding it under the Rails module is because then invoking the generator would be

```bash
bin/rails g component MyComponent
```

which can potentially create some confusion since it's not generating a "Rails component" but rather an "ActionView component". Having it under the ActionView module makes it explicit that the generator will creator an ActionView component.

We can move it under the Rails module if so desired.

Reply:
If I had to pick, I would defer to matching the syntax from ActionMailer, `rails g component FooComponent`. 

The less surprise, the better 😄.